api/src/main/java/org/grammatek/apis/UsersApi.kt

```diff
@@ -76,33 +82,34 @@ class DevelopersApi(basePath: kotlin.String = org.grammatek.apis.DevelopersApi.C
     }
 
     /**
-    * Correct grammar of text
     * 
-    * @param text  
-    * @return ApiResponse<YfirlesturResponse?>
+    * Correct spelling/grammar of text
+    * @param correctRequest  
+    * @return ApiResponse<CorrectResponse?>
     * @throws IllegalStateException If the request is not correctly configured
     * @throws IOException Rethrows the OkHttp execute method exception
     */
     @Suppress("UNCHECKED_CAST")
     @Throws(IllegalStateException::class, IOException::class)
-    fun correctApiPostWithHttpInfo(text: kotlin.String) : ApiResponse<YfirlesturResponse?> {
-        val localVariableConfig = correctApiPostRequestConfig(text = text)
+    fun correctApiPostWithHttpInfo(correctRequest: CorrectRequest) : ApiResponse<CorrectResponse?> {
```

Comment:
and this call then `postCorrectWithHttpInfo` ?

Reply:
I will look into how we can change that, I believe it is generated from the path of the API which is /correct.api

Reply:
ok